cli/main.go

```diff
@@ -62,6 +62,7 @@ var (
 	gid          = flag.Int("gid", -1, "User group id (gid)")
 	verbose      = flag.Int("verbose", 0, "Enable verbose debugging")
 	version      = flag.Bool("version", false, "Print the version and exit")
+	metadataDir  = flag.String("metadataDir", getDefaultMetadataDir(), "Directory to use for logfile and database. Defaults to $HOME/.dxfuse. Created if it does not exist.")
```

Comment:
I'd say `"Directory to use for dxfuse's internal metadata (log file, state database, etc). Created if does not exist. Defaults to " + getDefaultMetadataDir() + "."`

Reply:
Also, maybe stateFolder is a better name as metadata is also used in the context of DNAnexus file objects.